assets/src/edit-story/components/checklistCard/helpers/getCardType.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CARD_TYPE, GRID_VARIANT } from '../constants';
+
+/**
+ *
+ * @param {Object} props props passed to function
+ * @param {string} props.cardType type of card getting renderedText.Text
+ * @param {number} props.thumbnailCount how many thumbnails are getting rendered inside the card. Defaults to 0.
+ * @return {string} type of card.
+ */
+export const getCardType = ({ cardType, thumbnailCount }) => {
+  if (cardType === CARD_TYPE.SINGLE_ISSUE) {
+    return thumbnailCount > 0
+      ? GRID_VARIANT.SINGLE_WITH_THUMBNAIL
+      : GRID_VARIANT.SINGLE;
+  }
```

Comment:
pretty sure that for variants with more than 1 issue or for video optimization we can just through those qualifications in here and then just spit out new grid values in css!